src/main/java/com/github/iahrari/pricingservice/service/PriceServiceImpl.java

```diff
@@ -0,0 +1,70 @@
+package com.github.iahrari.pricingservice.service;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.github.iahrari.pricingservice.dto.Price;
+import com.github.iahrari.pricingservice.exception.PriceRequestFieldsException;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class PriceServiceImpl implements PriceService {
+    private final List<String> addresses = Arrays.asList(
+        "Vanak", "Azadi", "Tehran Pars", "Shohada"
+    );
+
+    @Override
+    public Price getPrice(String source, String destination) {
+        if(ifAnyEquals(source, destination, addresses.get(0)) 
```

Comment:
This block code is not clean and readable.
The switch case or strategy pattern is better than if-else